woocommerce/woocommerce-ios
# Remove order creation single-selection and feature flag

Closes: #9081 
Closes: #9452 


## Description
This PR addresses removing the legacy logic for product single-selection by:
- Removes feature flag from `DefaultFeatureFlagService`
- Removes the flag from `MockFeatureFlagService`
- Removes order creation single-selection logic
- Updates Unit Tests

## Testing instructions
There are no changes in the logic compared with the feature flag being enabled, but the following can be confirmed:
- Go to Orders > Tap `+` > Confirm that the string `+ Add Products` it's plural, then tap on it
- Products can be selected/unselected on a multi-selection basis, and tapping on "X products selected" does add them to the Order.
- On an Order with products, these can be removed  by tapping on `-` , and confirming by tapping on `Delete product from order`.
